client/python/gradio_client/client.py

```diff
@@ -659,15 +671,23 @@ def _render_endpoints_info(
 
         human_info = f"\n - predict({rendered_parameters}{final_param}) -> {rendered_return_values}\n"
         human_info += "    Parameters:\n"
-        if endpoints_info["parameters"]:
-            for info in endpoints_info["parameters"]:
+        if parameter_info:
+            for info in parameter_info:
                 desc = (
                     f" ({info['python_type']['description']})"
                     if info["python_type"].get("description")
                     else ""
                 )
+                default_value = info.get("parameter_default")
```

Comment:
I think we need to traverse the value here to only pull out the actual filepath or url

Reply:
So the view API page takes care of this automatically:

<img width="695" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/262c8d0c-5db5-4782-8a4a-cbe2237cdf1c">

but the `.view_api()` function does not

<img width="744" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/ad9cdc0b-f0f4-4818-80c5-ccfa8d0b5251">

```py
import gradio as gr

demo = gr.Interface(
    lambda x:x,
    gr.Audio("test.mp3"),
    "audio"
)

_, url, _ = demo.launch()

from gradio_client import Client

client = Client(url)

client.view_api()
```

Reply:
I'll fix in the `.view_api()`

Reply:
Yea sorry that's what I meant!